docs/examples/rbac/ruby/Makefile

```diff
@@ -1,3 +1,5 @@
+unexport BUNDLE_GEMFILE
```

Comment:
I saw this in another PR as well, but could you add an inline comment explaining what it's doing?

Reply:
Yeah good idea, this took me a while to figure out